test/facets/Token/behaviors/bridgeToken.t.sol

```diff
@@ -0,0 +1,198 @@
+    /// @dev It tests if bridging with zero amount should not be supported
+    function test_bridgingWithZeroAmountShouldNotBeSupported() public {
+        _enableChain(OWNER);
+
+        vm.expectRevert(TokenBridge__NoZeroAmount.selector);
+
+        vm.startPrank(ALICE);
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            0
+        );
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if bridging more than the balance should not be supported
+    function test_bridgingMoreThanBalanceShouldNotBeSupported() public {
+        _enableChain(OWNER);
+
+        vm.startPrank(ALICE);
+
+        vm.expectRevert(TokenBridge__InsufficientBalance.selector);
+
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            actualAliceBalance + 10
+        );
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if bridging with uint max transfers all the balance
+    function test_bridgingWithUintMaxShouldTransferAllBalance() public {
+        _enableChain(OWNER);
+
+        vm.startPrank(ALICE);
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            type(uint256).max
+        );
+
+        vm.assertEq(token.balanceOf(ALICE), 0);
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if bridging with a valid amount should transfer the correct amount
+    function test_bridgingShouldTransferTheCorrectAmount() public {
+        _enableChain(OWNER);
+
+        vm.startPrank(ALICE);
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            10 ether
+        );
+
+        vm.assertEq(token.balanceOf(ALICE), actualAliceBalance - 10 ether);
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if the AxelarGateway event is emitted properly
+    /// @dev It is the most important part since that event allows the bridging service to start
+    function test_bridgingShouldEmitAnAxelarEvent() public {
+        _enableChain(OWNER);
+
+        vm.startPrank(ALICE);
+        vm.expectEmit(true, true, true, true, ARBITRUM_AXELAR_GATEWAY);
+        emit ContractCall(
+            tokenAddress,
+            supportedChain,
+            destinationAddress,
+            keccak256(abi.encode(10 ether, ALICE)),
+            abi.encode(10 ether, ALICE)
+        );
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            10 ether
+        );
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It check if an internal event is emitted when initiating the bridge
+    function test_bridgingShouldEmitAnInternalEvent() public {
```

Comment:
This name would look like:
```suggestion
    function test_bridgeToken_emitsTokenBridgeInitialisedEvent() public {
```

Reply:
fixed here https://github.com/giovannifranchi/insrt-v2-contracts/commit/63acdaccf6b616e97578b7d62e1f167b6ac8d531